meson.build

```
@@ -6,22 +6,30 @@ project('DirectX-Headers', 'cpp', version : '1.610.0',
 
 inc_dirs = [include_directories('include', is_system : true)]
 install_inc_subdirs = ['']
+compile_options = []
 
 if host_machine.system() != 'windows'
     inc_dirs += include_directories('include/wsl/stubs', is_system : true)
     install_inc_subdirs += ['', 'wsl/stubs', 'directx']
+elif meson.get_compiler('cpp').get_id() == 'gcc'

```

Should the condition be something like the following? Because clang can also be used in mingw toolchain.

```suggestion
elif meson.get_compiler('cpp').get_id() != 'msvc'

```

---

Right, but Clang can also be used with the Windows SDK. Maybe this just needs to compile a test program instead...

---

Then checking like this `get_id != msvc && get_id != clang-cl` may be correct. Here is the list of compiler id https://mesonbuild.com/Reference-tables.html#compiler-ids

---

Yeah that'll probably work.